### DISCUSSION TITLE
Portugol Studio não vê arquivos em MacOS Catalina 10.15.4+

### DISCUSSION THREAD

**Discussion by AdsonEsteves** [UNIVALI-LITE/Portugol-Studio#886] [Problemas]
# Resumo
A partir da versão do MacOS Catalina 10.15.5 um novo sistema de segurança foi implementado. Com ele apenas aplicativos permitidos podem acessar as pastas do usuário.
Isso fez com que o Portugol Studio não conseguisse mais ver os arquivos e pastas do sistema para abri-los.

<img src="https://user-images.githubusercontent.com/8836540/98746737-d2945300-2394-11eb-93e6-326ab1afda83.png" width="480">

Por enquanto não sabemos como corrigir o problema pelo Portugol, mas existe uma maneira de fazer esses arquivos aparecerem novamente se o usuário garantir essa permissão ao Portugol Studio.
Basta garantir acesso completo ao disco ao aplicativo do Portugol Studio, através das configurações de privacidade.

O seguinte tópico explicará como resolve o problema

# Usuários
Aqui vamos mostrar passo a passo como dar permissão de acesso ao Portugol Studio.

1 - Abra `Preferencias do Sistema` > `Segurança e Privacidade`

<img src="https://user-images.githubusercontent.com/8836540/98867686-c4563d80-244d-11eb-9beb-dba0060cbdad.png" width="480">

2 - Seleciona a aba `Privacidade`

<img src="https://user-images.githubusercontent.com/8836540/98867592-94a73580-244d-11eb-9f36-1042dae0cdf8.png" width="480">

3 - Selecione na lista a opção `Acesso total ao disco` e clique no cadeado abaixo

<img src="https://user-images.githubusercontent.com/8836540/98867405-3e39f700-244d-11eb-9449-d08e01b8520e.png" width="480">

4 - Adicione as credenciais de administrador e clique em desbloquear

<img src="https://user-images.githubusercontent.com/8836540/98867476-5ad62f00-244d-11eb-9c21-6e86127c1747.png" width="480">

5 - Clique no botão `+`

<img src="https://user-images.githubusercontent.com/8836540/98868464-ec926c00-244e-11eb-99f4-c453b180e5b2.png" width="480">

6 - Selecione o aplicativo do Portugol Studio

<img src="https://user-images.githubusercontent.com/8836540/98867508-6d506880-244d-11eb-865e-6443a4d1d3d5.png" width="480">
<img src="https://user-images.githubusercontent.com/8836540/98867555-82c59280-244d-11eb-8b55-45c32355270a.png" width="480">

7 - Reinicie o Portugol e agora ele deve ter acesso as pastas e será possível ver os arquivos .por
<img src="https://user-images.githubusercontent.com/8836540/98747383-2a7f8980-2396-11eb-93ad-9028b5b5e9ee.png" width="480">

Caso possuam alguma dúvida ainda ou sugestão, deixem nos comentários.

| (• ◡•)| 39 (❍ᴥ❍ʋ)

**Comment by pitohuiJuh** [comment]
Obrigada pela ajuda! Muito útil!

**Comment by anita-falchi** [comment]
super obrigada!

**Comment by Gracinhaflor** [comment]
olá, ja instalei e desinstalei a versao ( 2.7.5 windows.xe ) diversas vezes, meu windows é 10 e o software instala incompleto ,faltando as bibliotecas ,a parte onde analisa as variaveis ,vetores,o que dificulta exercitar sem esses recursos, no video sobre instalaçao que vi ,mostra que o windows defender é quem desativa essas funcoes,mas pior que no momento que estou instalando nao aparece pra mim durante a instalaçao a mensagem do windows bloqueando ,para que eu possa interferir!! se alguem souber como resolver esse problema me ajude por favor.
